README.rst

```diff
@@ -134,6 +134,31 @@ Basic usage
     a_view.an_input.is_displayed
 
 
+``__locator__()`` and ``__element__()`` protocol
+------------------------------------------------
+
+To ensure good structure, a protocol of two methods was introduced. Let's talk a bit about them.
+
+``__locator__()`` method is not implemented by default on ``Widget`` class. Its sole purpose is to
```

Comment:
It is indeed not. Widget does not contains `__locator__` by default. You either specify ROOT or define it yourself. :smiley: